starboard/nplb/player_test_fixture.cc

```diff
@@ -281,15 +289,8 @@ void SbPlayerTestFixture::TearDownPlayer() {
   ASSERT_TRUE(HasReceivedPlayerState(kSbPlayerStateDestroyed));
   ASSERT_FALSE(error_occurred_);
 
-  // Reset internal states.
   player_ = kSbPlayerInvalid;
```

Comment:
nit: move them to the ifs above that destroys them.

Reply:
The |player_| pointer might be used to check the player events sent after calling SbPlayerDestroy but before SbPlayerDestroy is finished, so we need to reset the pointer at the end of the function after player events and state checks.